stable/instana-agent/README.md

```diff
@@ -116,6 +116,7 @@ The following table lists the configurable parameters of the Instana chart and t
 | `rbac.create`                      | Whether RBAC resources should be created                                | `true`                                       |
 | `serviceAccount.create`            | Whether a ServiceAccount should be created                              | `true`                                       |
 | `serviceAccount.name`              | Name of the ServiceAccount to use                                       | `instana-agent`                              |
+| `kubernetes.clusterName`           | Display name of the monitored Kubernetes Cluster (overrides zone.name)  | `nil`                                        |
```

Comment:
I'm a bit divided by using `kubernetes` for naming the variable, as HELM charts should work with OpenShift as well. Until now, we only used "kubernetes" for documentation, nothing that cannot easily be changed. Unfortunately can't directly provide a better alternative though...
Perhaps any ideas from others?

Reply:
Yea I thought about this. I guess Kubernetes is the upstream project for all the other ones. We could make it more generic like `cluster.name`?

Reply:
Yeah, I like `cluster.name`.

Should we also add an explanation of the differences between `zone.name` and this new configuration? I'm not sure if people could get confused by which one they need to set?

Reply:
`cluster.name` sounds good to me as well.